features/schemas/enable.json

```diff
@@ -0,0 +1,66 @@
+{
+  "type": "object",
+  "properties": {
+    "_schema_version": {
+       "type": "number"
+    },
+    "result": {
+       "type": "string"
+    },
+    "errors": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "required": [ "message", "service", "type" ],
+        "properties": {
+          "message": {
+            "type": "string"
+           },
+           "service": {
+             "type": ["null", "string"]
+           }
+        },
+        "patternProperties": {
+           "^type$": {
+             "type": "string"
+          }
```

Comment:
interesting, I haven't seen this before. Is this necessary to have a property with name "type"?

Reply:
We can make this an enum as well

Reply:
Yes, it is a hacky approach I found to allow a field with the name `type`